miniwallet/package.json

```diff
@@ -16,15 +16,16 @@
         "check": "npx hardhat check",
         "compile": "npx hardhat compile",
         "coverage": "npx hardhat coverage",
-        "deploy": "npx hardhat deploy",
+        "deploy": "npx hardhat deploy --tags deploy",
+        "deploy-upgrade": "npx hardhat deploy --tags upgrade",
         "hardhat-local": "npx hardhat node --no-deploy",
         "flat": "npx hardhat flatten ./contracts/MiniWallet.sol >> ./flat/MiniWallet.sol",
         "gas": "REPORT_GAS=true npx hardhat test",
         "test": "npx hardhat test"
     },
     "devDependencies": {
         "@atixlabs/hardhat-time-n-mine": "^0.0.5",
-        "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers",
+        "@nomiclabs/hardhat-ethers": "yarn:hardhat-deploy-ethers",
```

Comment:
what is this version?

Reply:
it is used for [hardhat deploy](https://github.com/wighawag/hardhat-deploy#npm-install-hardhat-deploy)

```
if you use ethers.js we recommend you also install hardhat-deploy-ethers which add extra features to access deployments as ethers contract.

Since hardhat-deploy-ethers is a fork of @nomiclabs/hardhat-ethers and that other plugin might have an hardcoded dependency on @nomiclabs/hardhat-ethers the best way to install hardhat-deploy-ethers and ensure compatibility is the following:

npm install --save-dev  @nomiclabs/hardhat-ethers@npm:hardhat-deploy-ethers ethers
Which means you then need to do require("@nomiclabs/hardhat-ethers") instead of require("hardhat-deploy-ethers") in your hardhat.config.js file.

More details on hardhat-deploy-ethers repo : https://github.com/wighawag/hardhat-deploy-ethers#readme
```